applications/openshift/risk-assessment/container_security_operator_exists/tests/ocp4/e2e-remediation.sh

```diff
@@ -14,3 +14,8 @@ done
 echo "waiting for container-security-operator deployment to be ready"
 oc wait -nopenshift-operators --for=condition=Available  --timeout=300s \
     deployment/container-security-operator
+
+echo "waiting the subscription to have .status.installedCSV"
+while [ -z "$(oc get subscription container-security-operator -nopenshift-operators -o jsonpath='{.status.installedCSV}')" ]; do
+    sleep 3
+done
```

Comment:
Please, ensure the files have proper ending. Most of the tests failed because of this.

Reply:
I think we just need to remove the newline here and we should be good?

```
/__w/content/content/applications/openshift/risk-assessment/container_security_operator_exists/tests/ocp4/e2e-remediation.sh
1 of 10900 files do not have the correct ending.
```